cli/command/formatter/stack.go

```diff
@@ -17,16 +21,14 @@ type Stack struct {
 	Name string
 	// Services is the number of the services
 	Services int
-	// Orchestratort is the platform on which the stack is deployed
+	// Orchestrator is the platform where the stack is deployed
 	Orchestrator string
+	// Namespace is the Kubernetes namespace assigned to the stack
+	Namespace string
 }
 
 // NewStackFormat returns a format for use with a stack Context
 func NewStackFormat(source string) Format {
-	switch source {
-	case TableFormatKey:
```

Comment:
This is a change in behaviour; before this change:

```bash
docker stack ls --format=table
NAME                SERVICES
foobar              1
```

After this change:

```bash
docker stack ls --format=table

# (no output)
```


Perhaps remove this function, and in swarm/kubernetes do;

```go
if len(format) == 0 || format == formatter.TableFormatKey {
	format = formatter.KubernetesStackTableFormat
}

stackCtx := formatter.Context{
	Output: dockerCli.Out(),
	Format: formatter.Format(format),
}
```